op-node/rollup/types.go

```diff
@@ -307,10 +333,14 @@ func (c *Config) Description(l2Chains map[string]string) string {
 	if l2Chains != nil {
 		networkL2 = l2Chains[c.L2ChainID.String()]
 	}
+	// replace using opBNB networks
+	networkL2 = NetworkNames[c.L2ChainID.String()]
```

Comment:
There is also a lint here, due to the assignment on this line, which makes the assignment on line 332 redundant. It is recommended to modify line 332 to: `var networkL2 string`
In fact, 337 makes 334 redundant. The entire if statement of 334 may be deleted.